tests/create_attestee.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+# A simple helper for creating an environment for the receiver of the newly produced certification
+
+ATTESTEE_DIR=/tmp/gks-attestee
+
+rm -rf "${ATTESTEE_DIR}.bak"
+mv -f "$ATTESTEE_DIR" "${ATTESTEE_DIR}.bak"
```

Comment:
```suggestion
if [[ -d $ATTESTEE_DIR ]]; then
  mv -f "$ATTESTEE_DIR" "${ATTESTEE_DIR}.bak"
fi
```

In this way you don't need to create an empty directory that will be renamed to `.bak`.